client/dashboard/utils/site-url.ts

```diff
@@ -1,8 +1,13 @@
 import type { Site } from '../data/types';
 
-export function getSiteHostname( site: Site ) {
-	if ( site.URL === '' ) {
-		return site.URL;
-	}
-	return new URL( site.URL ).hostname;
+/**
+ * Returns the site's URL without the protocol.
+ *
+ * This is useful for displaying the site's URL in a user-friendly way.
+ * Note that we cannot just return the URL's hostname, because the URL
+ * could contain the path to the site's subdirectory in case of multi-site
+ * installations.
```

Comment:
> The code is so self-explanatory that it's removing the protocol

I think that's the point. The code is so simple that it misleads the reader on its purpose IMO. The function is not about removing the protocol at all (that is just the current implementation).

```suggestion
 * Returns a user-friendly version of the site's URL.
 *
 * This not only produces a clean version of the site's domain, but
 * also deals with the case of Jetpack multi-sites using subdirectory
 * installations.
```

Reply:
Thanks, applied the suggestion :)